FinniversKit/Sources/Components/KeyValueGridView/KeyValueGridView.swift

```diff
@@ -80,11 +80,41 @@ public class KeyValueGridView: UIView {
     }
 
     private func createCellView(for pair: KeyValuePair) -> UIView {
-        let stackView = UIStackView(axis: .vertical, spacing: Warp.Spacing.spacing25, alignment: .leading, distribution: .equalSpacing, withAutoLayout: true)
+        let stackView = UIStackView(
+            axis: .vertical,
+            spacing: Warp.Spacing.spacing25,
+            alignment: .leading,
+            distribution: .equalSpacing,
+            withAutoLayout: true
+        )
 
         let titleLabel = Label(style: titleStyle, numberOfLines: 2, withAutoLayout: true)
         titleLabel.lineBreakMode = .byWordWrapping
 
+        let titleContainer = UIStackView()
+        titleContainer.axis = .horizontal
+        titleContainer.alignment = .center
+        titleContainer.distribution = .fill
+        titleContainer.spacing = Warp.Spacing.spacing100
+
+        titleLabel.text = pair.title
+
+        titleContainer.addArrangedSubview(titleLabel)
+        if let infoText = pair.infoTooltip, !infoText.isEmpty {
+            let infoButton = UIButton(type: .custom)
+            infoButton.setImage(Warp.Icon.info.uiImage, for: .normal)
+            infoButton.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
There's no label for VoiceOver users for this button. How should the button and the presentation work for them?